jsonrpc/endpoints_zkevm.go

```diff
@@ -266,3 +266,22 @@ func (z *ZKEVMEndpoints) GetFullBlockByHash(hash types.ArgHash, fullTx bool) (in
 		return rpcBlock, nil
 	})
 }
+
+// GetNativeBlockHashesInRange return the state root for the blocks in range
+func (z *ZKEVMEndpoints) GetNativeBlockHashesInRange(filter NativeBlockHashBlockRangeFilter) (interface{}, types.Error) {
+	return z.txMan.NewDbTxScope(z.state, func(ctx context.Context, dbTx pgx.Tx) (interface{}, types.Error) {
+		fromBlockNumber, toBlockNumber, rpcErr := filter.GetNumericBlockNumbers(ctx, z.cfg, z.state, z.etherman, dbTx)
+		if rpcErr != nil {
+			return nil, rpcErr
+		}
+
+		nativeBlockHashes, err := z.state.GetNativeBlockHashesInRange(ctx, fromBlockNumber, toBlockNumber, dbTx)
```

Comment:
If toBlockNumber doesn't exist it returns less values than expected. The same if fromBlockNumber doesnt exists.

 I don't known if it's the desired behaviour. Maybe it needs a comments some place

Reply:
What do you mean by doesn't exist? Can you please provide an example?

Reply:
I have l2block from 1-200 and I ask for 100-300, It'll return 100 hashes